### DISCUSSION TITLE
How to capture `ctrl+enter` or `shift+enter`?

### DISCUSSION THREAD

**Discussion by rohanshiva** [charmbracelet/bubbletea#609] [Q&A] [answered]
Is it possible to capture `ctrl+enter` or `shift+enter`?

**Comment by meowgorithm** [comment, marked as answer]
Unfortunately, no. This is a limitation on the terminal level.

**Reply by rohanshiva** [reply]
Thanks!

**Reply by aymanbagabas** [reply]
Update: @rohanshiva We're working on Bubble Tea v2 which will support enhanced keyboard features that allow you to detect these key events. You could try out the[ alpha.1](https://github.com/charmbracelet/bubbletea/discussions/1156) version.
